<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLbl</class>
 <widget class="QWidget" name="FormLbl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Этикетка</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Марка</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="DbComboBox" name="comboBoxMar">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Диаметр</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="DbComboBox" name="comboBoxDiam">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Намотка</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="DbComboBox" name="comboBoxPack">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Партия</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEditPart"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Плавка</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEditPlav"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Дата партии</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="3">
            <widget class="DbDateEdit" name="dateEditPart">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Ориг. партия</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditOrigPart">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>год</string>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="lineEditYear">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Намотчик</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="3">
            <widget class="QComboBox" name="comboBoxNam">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Масса нетто, кг</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEditMas"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>156</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonLbl1">
           <property name="text">
            <string>Этикетка (46*66,7)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Настройки</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Волочильщики</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="DbViewer" name="tableViewVol"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Намотчики</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="DbViewer" name="tableViewNam"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>ОТК</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="DbViewer" name="tableViewOtk"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>db/dbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>DbDateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbdateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxMar</tabstop>
  <tabstop>comboBoxDiam</tabstop>
  <tabstop>comboBoxPack</tabstop>
  <tabstop>lineEditPart</tabstop>
  <tabstop>lineEditPlav</tabstop>
  <tabstop>dateEditPart</tabstop>
  <tabstop>lineEditOrigPart</tabstop>
  <tabstop>lineEditYear</tabstop>
  <tabstop>comboBoxNam</tabstop>
  <tabstop>lineEditMas</tabstop>
  <tabstop>pushButtonLbl1</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableViewOtk</tabstop>
  <tabstop>tableViewNam</tabstop>
  <tabstop>tableViewVol</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
